templates/gitconfig.epp

```diff
@@ -0,0 +1,10 @@
+[user]
+  name = <%= $git_username %>
```

Comment:
@josephoaks Where is the `git_username` variable set? It would be ideal to pass it in as an argument to the `epp` function in the calling class.

Reply:
I just noticed that the custom fact didn't get pushed in there for some reason